app/src/main/java/com/codesoom/assignment/ResponseHandler.java

```diff
@@ -0,0 +1,83 @@
+package com.codesoom.assignment;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+import static com.codesoom.assignment.JSONParser.*;
+import static com.codesoom.assignment.TaskManager.*;
+
+/**
+ * handle responses content by requests
+ * TODO: if possible, change switch statement to hashmap
+ */
+public class ResponseHandler {
+    public String handle(String method, String path, List<Task> tasks, String body) throws IOException, ResponseHandlingException {
+        // check wrong path
+        if (!path.matches("/tasks/*[0-9]*")) {
+            throw new ResponseHandlingException(ResponseHandlingException.ErrorCode.WRONG_PATH);
+        }
+
+        Long taskId = extractTaskId(path);
+
+        switch (method) {
+            case "GET":
+                // fetch task list
+                if (path.equals("/tasks")) {
+                    return tasksToJSON(tasks);
+                }
+
+                // fetch a task
+                Task selectedTask = getTask(tasks, taskId);
+                // TODO: getTask에서 아래의 로직을 처리할 수 있도록 함
+                if (selectedTask.getId() == null) {
+                    throw new ResponseHandlingException(ResponseHandlingException.ErrorCode.WRONG_PATH);
+                }
+
+                return taskToJSON(selectedTask);
+
+            case "POST":
+                if (body.isBlank()) break;
+
+                Task newTask = toTask(body, (long) (tasks.size() + 1));
+                tasks.add(newTask);
+                return taskToJSON(tasks.get(tasks.size() - 1));
+
+            case "PUT": case "PATCH":
```

Comment:
`case`를 한 줄로 모으지 않는 것이 좋습니다.
```suggestion
            case "PUT":
            case "PATCH":
```